Sources/Build/TestObservation.swift

```diff
@@ -132,6 +132,8 @@ public func generateTestObservationCode(buildParameters: BuildParameters) -> Str
         @_exported import WinSDK
         #elseif os(WASI)
         @_exported import WASILibc
+        #elseif canImport(Android)
+        @_exported import Android
```

Comment:
This one potentially as well - the Windows case is probably for additional Win32 APIs that are needed, but I am unsure about the re-export there 🤔.

Reply:
Done